### PR TITLE
Reset `table-layout` to `auto`

### PR DESCRIPTION
**Pull request by tombye**
This removes the `table-layout: fixed` style for tables in tech' docs content, introduced as part of https://github.com/alphagov/tech-docs-gem/commit/ffa22615851866bb7052a87285fca30273f0ac54. 

## Why did we fix our table layouts?

It was introduced as part of a pull request that made 2 changes:
1. the `max-width: 40em` for `<pre>` blocks was removed to stop [issue #100](https://github.com/alphagov/tech-docs-gem/issues/100)
2. `table-layout: fixed` was added to fix the problems `max-width: 40em` solved

## Why did we constrain the width of our `<pre>` tags?

The comments above `max-width: 40em` mention `<pre>` tags pushing tables to be larger than the main column (see https://github.com/alphagov/tech-docs-gem/commit/ffa22615851866bb7052a87285fca30273f0ac54#diff-d7f12a4697fd2195a8c62339caee5c77L122).

### 1. It causes an issue with existing tables

It created [issue #133](https://github.com/alphagov/tech-docs-gem/issues/133) with Notify's tech' docs.

### 2. Removing it shouldn't effect any other existing tables

I went through the main users of this gem and found all the tables in them (thanks @nickcolley for their details):

https://docs.google.com/spreadsheets/d/1OZGQifZIw0I2GXqmrp78DoI3R2sfcxn5IcFq3Y5RXqA/edit#gid=0

I can't find any tables that use `<pre>` tags in table cells, just inline `<code>` blocks.

I also tested all these tables without `table-layout: fixed` and none of them displayed in ways that distorted the content or made the table wider than the main column.

### 3. Tables that are too wide don't break the main column

Tables are wrapped in a fixed-width container which scrolls. This means that if they're wider than the main column they don't break it and users can scroll to access their content. For example:

https://www.docs.verify.service.gov.uk/legacy/build-ms/msa/#msa-tls-certificates

### We probably shouldn't be using block-level code in tables

Note: this point is just my opinion 🙂.

Middleman's markdown parser [Redcarpet uses PHP-Markdown style tables](https://github.com/vmg/redcarpet#and-its-like-really-simple-to-use. In its docs, PHP-Markdown says it allows 'span level formatting' in its tables:

https://michelf.ca/projects/php-markdown/extra/#table

There's no mention of official support for block-level formatting, like `<pre>`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
+
+Remove the `table-layout: fixed` style from the tables in tech' docs content, mainly due to it
```
Comment (nickcolley) [inline CHANGELOG.md]: Not blocking by ideally the changelog should say what is fixed for users, not the details of what we've been doing internally.

Comment (tombye) [reply CHANGELOG.md]: Makes sense. Does [this change](https://github.com/alphagov/tech-docs-gem/pull/154/commits/1f90f78c00bbe7f1426aacc476dcde1325284db3) help?

Comment (nickcolley) [reply CHANGELOG.md]: Great, I'd prefer if it didnt require looking in the link to understand the summary but good enough 👍 Thanks Tom

Comment (tombye) [reply CHANGELOG.md]: That's a good point actually. I've rewritten it in [this commit](https://github.com/alphagov/tech-docs-gem/pull/154/commits/a4a20f19eaf412c8e455768f974eb824f1374d8d).
